go/BUILD.bazel

```diff
@@ -0,0 +1,64 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
+load("//go:rules.bzl", "swig_native_go_library")
+
+swig_native_go_library(
+    name = "typedb_driver_go_native",
+    lib = "//c:typedb_driver_clib_headers",
+    package_name= "typedb_driver",
+    interface = "//c:typedb_driver.i",
+    includes = ["//c:swig/typedb_driver_go.swg"],
+    enable_cxx = True,
+    visibility = ["//visibility:public"],
+)
+
+go_library(
+    name = "driver-go",
+    srcs = glob(["*.go"]),
+    data = ["//:LICENSE"],
+    importpath = "typedb_driver/go",
+
+    clinkopts = [
+        "-L/Users/nickan/typedb-driver/bazel-bin/go", # TODO use relative path. For now use the bazel output path.
```

Comment:
Gonna rename it to `"-L/Users/*your_username*/typedb-driver/bazel-bin/go", leave the `TODO` as is